oeps/architectural-decisions/oep-XXXX-micro-frontend-domains.rst

```diff
@@ -0,0 +1,171 @@
+================================
+OEP-XXXX: Micro-frontend Domains
+================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-XXXX <oep-XXXX-micro-frontend-domains>`
+   * - Title
+     - Micro-frontend Domains
+   * - Last Modified
+     - 2022-11-09
+   * - Authors
+     - Adolfo R. Brandes <adolfo@tcril.org>
+   * - Arbiter
+     - TBD
```

Comment:
@arbrandes Feel free to list me (astankiewicz@2u.com) as an arbiter for this OEP. I'll brush up on the responsibilities of the arbiter role [here](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#step-3-review-with-arbiter).